advisories/1.0.6/BRSA-bjyw3iarh4yj.toml

```
@@ -0,0 +1,17 @@
+[advisory]
+id = "BRSA-bjyw3iarh4yj"
+title = "kernel CVE-2024-56751"
+cve = "CVE-2024-56751"
+severity = "moderate"
+description = "In the Linux kernel, the following vulnerability has been resolved: ipv6: release nexthop on device removal The CI is hitting some aperiodic hangup at device removal time in the"
```

```suggestion
description = "In the Linux kernel, the following vulnerability has been resolved: ipv6: release nexthop on device removal"
```

see https://github.com/torvalds/linux/commit/eb02688c5c45c3e7af7e71f036a7144f5639cbfe

---

Fixed the description with this recommendation